pages/glossary/changes-in-content.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Changes in content
+key: changes-in-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - Audio output
+  - CSS styling
+  - DOM tree
+---
+
+A _change in the content_ of a [web page][] caused by an event occurs when, by comparing the [web page][] before and after the event firing, at least one of the following occurs:
+
+- **visible changes:** the rendered pixels change in any part of the document that is currently within the [viewport][] or that can be brought into the [viewport][] via scrolling; or
+- **accessibility tree changes:** any state, property or event of a node representing an [accessible object][] of the [accessibility tree][] changes, or any node is inserted in, or removed from the [accessibility tree][]; or
+- **audible changes:** the audio reproduced by the [web page][] changes.
+
+If the [web page][] is reproducing time-based media, rendered pixels and reproduced audio will be changing as part of the playback. The comparison in this instance should compare the pixels and audio that are rendered and reproduced if the event is not fired, with the ones that are rendered if the event is fired.
+
+For the purpose of this definition, it is assumed that the event causing the changes is the only origin of changes to the [web page][] and therefore the page does not change if there is no such event.
+
+For the purpose of this definition, it is assumed that the changes happen instantly after the event firing and therefore the comparison between the page before and after the event firing can be made at any time after the firing of the event.
```

Comment:
I don't know whether this assumption will hold in practice (due to either micro-delays that can happen, or design choice to delay a bit the reaction).
But I guess it is OK given that this is a manual check and that I guess testers won't look at the result within the next micro-seconds…

Reply:
That was my reasoning. This is going to be a manual check, therefore we don't really need to concern oursevels with micro-delays.